diff --git a/node_modules/metro/src/lib/TerminalReporter.js b/node_modules/metro/src/lib/TerminalReporter.js
index b30e0cb..35cea2a 100644
--- a/node_modules/metro/src/lib/TerminalReporter.js
+++ b/node_modules/metro/src/lib/TerminalReporter.js
@@ -136,21 +136,17 @@ class TerminalReporter {
   }
   _logInitializing(port, hasReducedPerformance) {
     const logo = [
-      "",
-      "                        ▒▒▓▓▓▓▒▒",
-      "                     ▒▓▓▓▒▒░░▒▒▓▓▓▒",
-      "                  ▒▓▓▓▓░░░▒▒▒▒░░░▓▓▓▓▒",
-      "                 ▓▓▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▒▒▒▓▓",
-      "                 ▓▓░░░░░▒▓▓▓▓▓▓▒░░░░░▓▓",
-      "                 ▓▓░░▓▓▒░░░▒▒░░░▒▓▒░░▓▓",
-      "                 ▓▓░░▓▓▓▓▓▒▒▒▒▓▓▓▓▒░░▓▓",
-      "                 ▓▓░░▓▓▓▓▓▓▓▓▓▓▓▓▓▒░░▓▓",
-      "                 ▓▓▒░░▒▒▓▓▓▓▓▓▓▓▒░░░▒▓▓",
-      "                  ▒▓▓▓▒░░░▒▓▓▒░░░▒▓▓▓▒",
-      "                     ▒▓▓▓▒░░░░▒▓▓▓▒",
-      "                        ▒▒▓▓▓▓▒▒",
-      "",
-      "",
+      "                       .-'''-.             .-'''-." ,
+      "                      I .===. I           I .===. I",
+      "                      >< ô ô ><           >< ô ô ><",
+      "                     ( <_ . _> )         ( <_ . _> )",
+      "                      |-  A  -|           |-     -|",
+      "                  ====|-  N  -|====   ====|-  T  -|====",
+      "                      |-  D  -|           |-  R  -|",
+      "                      |-  R  -|           |-  A  -|",
+      `                      [-  E  -]           [-  N  -]`,
+      `                      [-  W  -]           [-     -]`,
+      "                      (__| |__)           (__| |__)",
     ];
     const color = hasReducedPerformance ? chalk.red : chalk.blue;
     this.terminal.log(color(logo.join("\n")));
